README.rst

```diff
@@ -107,8 +107,10 @@ and see the created barcode. That's it.
 
 Commandline::
 
-    $ python-barcode create "My Text" outfile
+    `$ python-barcode create "123456789000" outfile -b ean --text "text to appear under barcode" `
     New barcode saved as outfile.svg.
+    
+    # This may not work if `ImageWriter` is None 
```

Comment:
This clarification here doesn't really make sense to me. What are you trying to explain here?

Reply:
I mainly add this because in the previous code example mentioned in the doc,
```
$ python-barcode create -t png "My Text" outfile
New barcode saved as outfile.png.
```
didn't work for me because of this error `python-barcode: error: unrecognized arguments: -t`

when i looked into the source code I found this 
```python
# pybarcode.py

create_parser.add_argument(
        '--text', help='Text to show under the '
        'barcode.'
    )
    if ImageWriter is not None:
        create_parser.add_argument(
            '-t', '--type', help='Type of output '
            '[default: %(default)s].'
        )
    list_parser = subparsers.add_parser(
        'list', help='List available '
        'image and code types.'
    )
```
according to the code, the option `-t` will only be available if `ImageWriter` is not None, otherwise option `-t` will not be available and will throw error that the `-t` option is not recognized.

However, I haven't actually looked into the source code to understand ImageWriter just yet.

Reply:
Oh. `ImageWriter` is unavailable if Pillow is not installed (it's required for exporting to images instead of SVG).

It's probably best to state that